java/src/main/java/vw/jni/NativeUtils.java

```diff
@@ -88,7 +84,7 @@ else if (osName.toLowerCase().contains("linux")) {
             if (version == null) {
```

Comment:
some linux distros don't have a period in the version, such as debian:jessie.  If we throw an error, we won't be able to one day use this code in Debian.  Are you sure you want to add it?